Block/Adminhtml/System/Config/Form/Version.php

```diff
@@ -0,0 +1,49 @@
+<?php
+declare(strict_types=1);
```

Comment:
- We do not use strict_types directive.
- Please add copyright here, the same way as it added in other files in this repo.

Reply:
@vitaliyreznikov Done, pls check.